crawler.go

```diff
@@ -139,7 +158,8 @@ func (c crawler) exportaPlanilha(ctx context.Context, fName string) error {
 		chromedp.Click(`//*[@id="sc_btgp_btn_group_1_top"]`, chromedp.BySearch, chromedp.NodeVisible),
 		chromedp.Sleep(c.timeBetweenSteps),
 	); err != nil {
-		return fmt.Errorf("planilha não disponível: %v", err)
+		dataUnavailableErr("botão de download\n").Error()
+		os.Exit(STATUS_DATA_UNAVAILABLE)
```

Comment:
```suggestion
		return &dataUnavailableErr{"botão de download"}
```

@joaolgm poderia melhorar essa mensagem? Eu não entendi bem.